catalog/global-pasture-watch/projects_global-pasture-watch_assets_gsvh-30m_v1_short-veg-height_m

```diff
@@ -0,0 +1,110 @@
+local id = 'projects/global-pasture-watch/assets/gsvh-30m/v1/short-veg-height_m';
+local subdir = 'global-pasture-watch';
+local version = '1';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+local units = import 'units.libsonnet';
+local svh30m_v1 = importstr 'svh-30m_v1.md';
+local license = spdx.cc_by_4_0;
+
+local basename = std.strReplace(id, '/', '_');
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+{
+  id: id,
+  title: 'GPW Annual short vegetation height v' + version,
+  version: version,
+  'gee:status': 'beta',
+  description: |||
+    This dataset provides global median vegetation height
+    from 2000 at 30-m spatial resolution. 
+  ||| + svh30m_v1,
+  'gee:categories': ['plant-productivity'],
+  keywords: [
+    'canopy',
+    'vegetation',
+    'land',
+    'landcover',
+    'global',
+  ],
+
+  providers: [
+    ee.producer_provider('Land and Carbon Lab Global Pasture Watch', 'https://landcarbonlab.org/data/global-grassland-and-livestock-monitoring'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent_global('2000-01-01T00:00:00Z', null),
+  summaries: {
+    'eo:bands': [
+      {
+        name: 'height',
```

Comment:
please add  
'gee:units': units.meter,

(then you don't have to say (m) in the description)